apps/judicial-system/backend/src/app/modules/defendant/defendant.service.ts

```diff
@@ -305,7 +305,17 @@ export class DefendantService {
         return { delivered: true }
       })
       .catch((reason) => {
-        this.logger.error('Failed to update case with defendant', { reason })
+        const sanitizedReason = JSON.stringify(reason)
+          .replace(
+            /Participant with id: \d{10}/g,
+            'Participant with id: **********',
+          )
```

Comment:
Stripping national ids from error logs is not necessary as it is already being done elsewhere.